TesseractOCR/G8Tesseract.mm

```diff
@@ -733,6 +825,10 @@ - (UIImage *)imageWithBlocks:(NSArray *)blocks drawText:(BOOL)drawText threshold
 
 - (BOOL)recognize
 {
+    if (!self.isEngineConfigured) {
+        return NO;
```

Comment:
Can you log a warning here that the engine isn't configured properly? This is probably the point at which most users would want to know if the engine is configured or not.